blockchain/blockchain-wallet-v4-frontend
# feat(remoteConfig): add Firebase SDK

## Description (optional)
Adds Firebase Remote Config implementation using [Load new values for next startup](https://firebase.google.com/docs/remote-config/loading#strategy_3_load_new_values_for_next_startup) loading strategy. Should be a superior replacement for all existing feature flag management solutions.

This initial version supports only Boolean values.

## Testing Steps (optional)
Set `web_ff_super_app` to true in Firebase console then check Redux store for the value in `remoteConfig`. Be aware of [the SDK throttling](https://firebase.google.com/docs/remote-config/get-started?platform=web#throttling). In case if SDK request fails, you will still see default config in the Redux; however, instead of dispatching `remoteConfig/setConfig` - `remoteConfig/initializeFailed` will be dispatched.
